pkgs/by-name/aq/aqualung/package.nix

```diff
@@ -0,0 +1,187 @@
+{
+  lib,
+  stdenv,
+  autoreconfHook,
+  pkg-config,
+  fetchFromGitHub,
+  nix-update-script,
+
+  # Mandatory
+  gtk3,
+  libxml2,
+
+  # Optional
+  enableLadspa ? true,
+  lrdf,
+  enableCdda ? true,
+  libcdio,
+  libcdio-paranoia,
+  enableCddb ? true,
+  libcddb,
+  enableSrc ? true,
+  libsamplerate,
+  enableIfp ? false, # FIXME: find the right dependency if it exists in nixpkgs
+  enableLua ? true,
+  lua,
+
+  # File I/O
+  enableSndfile ? true,
+  libsndfile,
+  enableFlac ? true,
+  flac,
+  enableVorbis ? true,
+  libvorbis,
+  enableSpeex ? true,
+  liboggz,
+  speex,
+  enableMpeg ? true,
+  libmad,
+  enableLame ? true,
+  lame,
+  enableMod ? true,
+  libmodplug,
+  enableMpc ? true,
+  libmpcdec,
+  enableMac ? true,
+  monkeysAudio,
+  enableWavpack ? true,
+  wavpack,
+  enableLavc ? true,
+  ffmpeg,
+
+  # Output
+  enableSndio ? true,
+  sndio,
+  enableAlsa ? true,
+  alsa-lib,
+  enableJack ? true,
+  libjack2,
+  enablePulse ? true,
+  pulseaudio,
+}:
+stdenv.mkDerivation rec {
+  pname = "aqualung";
+  version = "2.0";
+  nativeBuildInputs = [
+    autoreconfHook
+    pkg-config
+  ];
+  buildInputs =
+    [
+      gtk3
+      libxml2
+    ]
+    ++ lib.optional enableLadspa lrdf
+    ++ lib.optionals enableCdda [
+      libcdio
+      libcdio-paranoia
+    ]
+    ++ lib.optional enableCddb libcddb
+    ++ lib.optional enableSrc libsamplerate
+    ++ lib.optional enableIfp (throw "iRiver iFP driver not available in nixpkgs")
```

Comment:
If we don't support this driver yet then I would suggest commenting out the option for now